package.json

```diff
@@ -3,6 +3,10 @@
   "version": "0.9.1",
   "description": "React carousel image gallery component with thumbnail and mobile support",
   "main": "./build/image-gallery.js",
+  "files": [
+    "build",
+    "styles/css"
+  ],
```

Comment:
Not sure why this is needed if it's important, please open a separate PR/issue and explain the purpose.

Reply:
You can read more about this feature here: https://docs.npmjs.com/files/package.json#files. From my point of view, it's important, because it helps to reduce the package-size and remove unrequired files.

Reply:
I see, then please also add `styles/scss` as some users import from that

Reply:
Oh, missed that one. Thank you! Gonna do that.